### ISSUE TITLE
Tweak "success" headline messaging (onboarding)

### ISSUE THREAD

**Issue by cassiemc** [mozilla/webmaker.org#1113]
"Good start!" messaging after users click the 'next' button feels patronizing. I think most people will realize they've simply only typed something and not done anything too complicated, or even 'real' making. I'd bet most people are used to filling out form fields online. So, what are really trying to say?

Can we tie it back to the concept of "one step closer to really making online" – or the idea @xmatthewx mentioned during demos today of "previewing a making experience"? Even getting rid of the exclamation mark would help, imo.

cc @edrushka @HPaulJohnson

**Comment by HPaulJohnson** [comment]
Good feedback @cassiemc 

Will riff on some better alternatives and post here soon.

**Comment by secretrobotron** [comment]
@HPaulJohnson @edrushka should we just pick something we already wrote up on https://studiomofo.etherpad.mozilla.org/splash-page-copy ?

**Comment by edrushka** [comment]
Can we get an approximate character limit, @cassiemc ?

**Comment by xmatthewx** [comment]
heading - 12 chars will fit on a single line mobile, 14 fit on a single line on desktop.
sentence - 100 characters cap will work

**Comment by HPaulJohnson** [comment]
Did a quick braindump of some alternatives in the etherpad. I don't know if "the one" is in this list but hopefully it inspires additional ideas. If you have anything to add, go for it. 

https://studiomofo.etherpad.mozilla.org/splash-page-copy (starts on line 76)

Otherwise, hopefully @edrushka and I can run through it in the a.m. and make a final decision on new headline.

**Comment by cassiemc** [comment]
I left some comments in the etherpad and a few additional thoughts. Some great directions!

**Comment by edrushka** [comment]
I vote Welcome to Webmaker. Noted in the pad.

**Comment by secretrobotron** [comment]
@HPaulJohnson @cassiemc sound good?

**Comment by cassiemc** [comment]
wfm!

**Comment by secretrobotron** [comment]
Thoughts? Looks a little cluttered now, imo.

![screenshot 2015-01-22 13 52 14](https://cloud.githubusercontent.com/assets/403231/5862223/f10100ec-a23d-11e4-9409-b6fc945f1262.png)

**Comment by cassiemc** [comment]
Yeah I agree. You could probably adjust the text or the text box width to fit it on one line.

**Comment by HPaulJohnson** [comment]
+1 to getting it all on one line.
